### PR TITLE
Lock our pg client down to 9.5.x to match what is used in production

### PR DESCRIPTION
**Pull request by jgedeon-osi**
[#fix]

Our development environment used pg client 9.5, pre-int, preview, etc used pg 9.5 however our production and test containers used pg client 9.4. This caused issues when running migrations in the integrated test environment so I have upgraded the clients to match.

### CODE REVIEW

File: Dockerfile.production [hunk 1]
```diff
@@ -1,6 +1,9 @@
 FROM ruby:2.4.0
 LABEL application=intake_api_prototype
-RUN apt-get update -qq && apt-get install -y build-essential libpq-dev postgresql-client
+RUN apt-get update -qq && apt-get install -y software-properties-common
```
Comment (ThomasOSI) [inline Dockerfile.production]: Squish the RUN layers, as appropriate

File: docker/test/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,9 @@
 FROM ruby:2.4.0
 LABEL application=intake_api_prototype
-RUN apt-get update -qq && apt-get install -y build-essential libpq-dev postgresql-client
+RUN apt-get update -qq && apt-get install -y software-properties-common
```
Comment (ThomasOSI) [inline docker/test/Dockerfile]: Squish the RUN layers, as appropriate

File: Dockerfile.production (hunk 1 above)
Comment (jgedeon-osi) [reply Dockerfile.production]: Done

File: docker/test/Dockerfile (hunk 2 above)
Comment (jgedeon-osi) [reply docker/test/Dockerfile]: Done

**Pull request merged**